pkg/osquery/runtime/runner.go

```diff
@@ -66,6 +66,13 @@ func New(k types.Knapsack, opts ...OsqueryInstanceOption) *Runner {
 }
 
 func (r *Runner) Run() error {
+	if r.knapsack.InModernStandby() {
+		r.slogger.Log(context.TODO(), slog.LevelInfo,
+			"modern standby detected, skipping osquery instance start",
+		)
+		return nil
```

Comment:
If we return nil, what happens when it wakes up?

Reply:
I'm not certain this is the right way or if we should just do some kind of wait loop, it's difficult to test. Currently all the osq runtime tests don't work on windows, so I'm trying to update those.